paasta_tools/cli/utils.py

```diff
@@ -490,16 +490,21 @@ def run_paasta_serviceinit(subcommand, master, service, instancename, cluster, *
         delta = "--delta %s" % kwargs['delta']
     else:
         delta = ''
-    command = 'ssh -A -n %s sudo paasta_serviceinit %s%s%s %s %s' % (
+    if 'stream' in kwargs and kwargs['stream']:
```

Comment:
any reason to hide the ``stream`` parameter in kwargs? What's special about it that it's not just a regular named parameter that's defaulting to ``False``?